bb8/src/internals.rs

```diff
@@ -1,6 +1,10 @@
+#[cfg(not(target_has_atomic = "64"))]
+use portable_atomic::AtomicU64;
```

Comment:
Nit: this import should go in the next block, with external crate imports.

Reply:
done